pyrasterframes/src/main/python/setup.py

```diff
@@ -116,15 +116,15 @@ def run(self):
     install_requires=[
         'pytz',
         'shapely',
-        'pyspark',
-        'numpy',
+        'pyspark==2.3.3',
```

Comment:
Is pinning this version really needed for setup / install? Is there a version range that we can use?

Reply:
Yes. If it goes to 2.4 then the tests will fail. RF isn't currently compatible with 2.4 (only found that out due to this situation).

Reply:
It would be better to have it not pinned, if we can. So `pyspark<2.4` is there a lower bound version, that we know of? `2.0`?

Reply:
Gonna do it and add issue to find out and enforce lower bound.

Reply:
See #155